pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -657,7 +657,7 @@ timedelta}, default 'raise'
         return ccalendar.get_day_of_year(self.year, self.month, self.day)
 
     @property
-    def week(self):
+    def week(self) -> int:
```

Comment:
Not sure if this is correct. hope that one of the developers will correct me if needed.

Reply:
this is correct

Reply:
thank you @jbrockmendel